libs/protocol-client/src/lib/client/types.ts

```diff
@@ -0,0 +1,50 @@
+export type ContributionItem = {
```

Comment:
I don't think we need this object. For something like contribution list the return type is the the result key of type `ListContributionsQuery`. The way we can type this in the protocol frontend is by using ListContributionsQuery['result']. Even if there is no explicit type in protocol types then we should be able to access the underlying key type we return